client/components/poll-form.html

```diff
@@ -7,7 +7,7 @@
         </div>
         <div class="form-group">
             <label>Questão</label>
-            <input type="text" name="question" class="form-control" placeholder="Sua questão" required>
+            <input type="text" name="question" class="form-control" placeholder="Sua questão" required pattern="[a-zA-Z0-9]+">
```

Comment:
Nice @nilemarbarcelos , but maybe here the spaces can be allowed.

In my mind, the question text could content a spaces, because it where the user create the asking.

But, I think is important there is the pattern, what you think to include the 'space' in regex pattern?